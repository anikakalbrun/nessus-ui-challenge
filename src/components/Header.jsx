import React from 'react';

import logo from '../logo.svg';

export default function Header() {
    return (
        <header className="App-header">
            <img src={logo} className="App-logo" alt="logo"/>
            <h1 className="App-title">Nessus UI Assignment</h1>
        </header>
    );
}