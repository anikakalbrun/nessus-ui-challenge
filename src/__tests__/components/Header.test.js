import React from 'react';
import Header from '../../components/Header';

let wrapper;

describe('Header', () => {

    it('renders correctly', () => {
        wrapper = renderer.create(<Header/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
