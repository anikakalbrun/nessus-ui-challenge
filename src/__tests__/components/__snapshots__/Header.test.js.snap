// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="App-title"
  >
    Nessus UI Assignment
  </h1>
</header>
`;
